DEVELOPMENT.md

```diff
@@ -0,0 +1,71 @@
+# Development
+
+Terraform provides helpful [Extending Terraform][1] documentation for best practices around writing provider code. This document provides some guidelines for working with this project.
+
+## Prerequisites:
+
+-   [Terraform][2] 0.10.x
+-   [Go][3] 1.15 (to build the provider plugin)
+-   A clone of this repository and the [\$GOPATH environment variable][7] set
+-   [tfplugindocs][8]
```

Comment:
Do we have instructions on how to install it? Not sure I managed to do it right last time.

Reply:
Installation should be a combination of a standard `go get/install` and having GOPATH/bin in your path? I was hoping to not go too detailed on how to install the dependencies and instead rely on their documentation